scripts/run_ansible_test

```diff
@@ -34,20 +34,21 @@ collection_path='/usr/share/ansible/collections/ansible_collections'
 case ${USE_VENV} in
     y|yes|true):
         ansible_test="${HOME}/test-python/bin/ansible-test"
-        collection_path="${HOME}.ansible/collections/ansible_collections"
+        collection_path="${HOME}/.ansible/collections/ansible_collections"
         ;;
 esac
 
-mkdir -p ${HOME}/code/ansible_collections/cifmw/general
-rsync -a --delete ${PROJECT_DIR}ci_framework/* ${HOME}/code/ansible_collections/cifmw/general/
-ln -fs ${collection_path}/* ${HOME}/code/ansible_collections/
+mkdir -p ${HOME}/code
+ansible-galaxy collection install git+file://${PROJECT_DIR} -p ${HOME}/code
 
 # Create/append the sanity exceptions for the current ansible version
 ansible_version=$(python3 -c "import ansible; print(ansible.__version__)" | sed 's/\.[^.]*$//')
-cat  "${HOME}/code/ansible_collections/cifmw/general/tests/sanity/ignore.txt" >> \
-    "${HOME}/code/ansible_collections/cifmw/general/tests/sanity/ignore-${ansible_version}.txt"
+cat  "${HOME}/code/ansible_collections/cifmw/general/ci_framework/tests/sanity/ignore.txt" >> \
+    "${HOME}/code/ansible_collections/cifmw/general/ci_framework/tests/sanity/ignore-${ansible_version}.txt"
 
 pushd ${HOME}/code/ansible_collections/cifmw/general
+cp -r ci_framework/tests .
```

Comment:
With the collection structure I doubt we still need to separate the framework in the ci_framework dir and create symlinks... What's the point of that right now? @cjeanner

Reply:
PS: If we still need to have the ci_framework directory copying the tests directory is the only solution, as ansible-test doesn't follow symlinks.

Reply:
Can't confirm that symlinks don't work, I tried :) I kept the tests inside ci_framework for this first version because I was not sure whether it would break anything else, but I think it would be better to move them on level up to follow the conventional collection structure

Reply:
We can do a follow-up in order to move tests in the repository.